### DISCUSSION TITLE
How can I airdrop runes to a large number of people?

### DISCUSSION THREAD

**Discussion by Peytony440** [ordinals/ord#3698] [General]
Hello, I want to airdrop runes to a large number of people?

What technique can I use directly on ord?

Split Runes ?

**Comment by cryptoni9n** [comment]
Hi Peytony440 - unfortunately, there is no way to do this with ord at this time.  [An issue](https://github.com/ordinals/ord/issues/3658) has been opened for this enhancement request and it is currently on the [project tracker](https://github.com/orgs/ordinals/projects/1).
